driver-legacy/src/main/com/mongodb/DB.java

```diff
@@ -566,6 +565,13 @@ ReadPreference getCommandReadPreference(final DBObject command, @Nullable final
         }
     }
 
+    Codec<DBObject> getDefaultDBObjectCodec() {
```

Comment:
Basically copied from DBCollection, but it can't be exactly the same so not really shareable.